functionary/ui/templates/socialaccount/connections.html

```diff
@@ -0,0 +1,43 @@
+{% extends "base.html" %}
+{% block title %}
+    Account Connections
+{% endblock title %}
+{% block content %}
```

Comment:
Rather than having separate sections for removing and adding social account associations I wonder if it makes more sense to just list all of the available providers (the ones actually enabled, not the entire list supported by allauth), with the ones currently associated up top and the ones not associated on the bottom.  Then each one has an "Add" or "Remove" button depending on its current status.  

I'm including a super rough "mock-up" in the form of a spreadsheet.

![image](https://user-images.githubusercontent.com/86254668/208712047-fc30776f-0c72-443d-af6f-1e466b54f2d2.png)

Reply:
Rewrote this page to be less confusing.